src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.js

```diff
@@ -51,29 +51,22 @@ const ImageRenderer = (props) => {
     ) : (
         <ShowContextMenuContext.Consumer>
             {({anchor, report, action, checkIfContextMenuActive}) => (
-                <AttachmentModal
-                    allowDownload
-                    reportID={report.reportID}
-                    source={source}
-                    isAuthTokenRequired={isAttachment}
-                    originalFileName={originalFileName}
+                <PressableWithoutFocus
+                    styles={[styles.noOutline, styles.alignItemsStart]}
+                    onPress={() => {
+                        const route = ROUTES.getReportAttachmentRoute(report.reportID, source);
+                        Navigation.navigate(route);
```

Comment:
This can be replaced by a call to `useNavigation` and then `navigation.navigate('Report_Attachment', {.reportID, source })` 

Everywhere else we use `Navigation.navigate` and `ROUTES` so I've opted for the same approach here, but this allows deep linking and link sharing and this feature might be unwanted